setup.py

```diff
@@ -15,7 +15,7 @@
         long_description = f.read()
 
     install_requirements = [
-        "vivarium>=2.0.0",
+        "vivarium>=2.1.0",
```

Comment:
Does this require vivarium 2.1.0 (i.e. is there some reason to believe it wouldn't be compatible with vivarium 2.0.1)? If not, we shouldn't touch this pin.